.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
We can ignore these files using a [local git ignore](https://help.github.com/articles/ignoring-files/#create-a-local-gitignore)